nmdc_automation/workflow_automation/wfutils.py

```diff
@@ -11,7 +11,17 @@
 import hashlib
 from linkml_runtime.dumpers import json_dumper
 
-
+# TODO: Berkley refactoring:
+#  The NmdcSchema class - responsible for creating workflow and data object records
+#  to be inserted into the NMDC database will need to be updated to generate Berkley-compatible
+#  Datageneration and WorkflowExecution records.
```

Comment:
Same comment here, DataGeneration is abstract.

Reply:
Updated TODO with this detail